src/main/rx/rx.h

```diff
@@ -76,6 +76,14 @@ extern const char rcChannelLetters[];
 
 extern int16_t rcData[MAX_SUPPORTED_RC_CHANNEL_COUNT];       // interval [1000;2000]
 
+typedef enum {
+    RX_CHANNEL_STABILITY_LOW = 0,
+    RX_CHANNEL_STABILITY_MEDIUM,
+    RX_CHANNEL_STABILITY_HIGH,
+} rxChannelStability_e;
+
+extern uint8_t rcChannelStability[MAX_SUPPORTED_RC_CHANNEL_COUNT];
```

Comment:
In some cases it may be better to represent channel stability as bitmask ... testing multiple channels (rc commands) would be faster/easier. maybe:
```
uint32_t rcChanelStableHigh, rcChanelStableMedium;
// rcChanelStableLow == ~(rcChanelStableHigh|rcChanelStableMedium);
```
an each bit representing one channel.